schemas/post.py

```
@@ -0,0 +1,135 @@
+from pydantic import BaseModel
+from datetime import datetime
+from typing import List
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)